sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetRebaseDatetimeSuite.scala

```diff
@@ -170,7 +173,8 @@ abstract class ParquetRebaseDatetimeSuite
       // By default we should fail to read ancient datetime values when parquet files don't
       // contain Spark version.
       "2_4_5" -> failInRead _,
-      "2_4_6" -> successInRead _).foreach { case (version, checkDefaultRead) =>
+      "2_4_6" -> successInRead _,
+      "3_2_2" -> successInRead _).foreach { case (version, checkDefaultRead) =>
```

Comment:
ditto, should be `3_2_1`?

Reply:
The generated files have `3.2.2` in metadata, in fact. I would keep it as is. For us, more important that the files have the keys:
```
extra:       org.apache.spark.legacyINT96 =
extra:       org.apache.spark.legacyDateTime =
```
but **not any time zone**.